src/algorithmForNN/fileUtils.py

```diff
@@ -12,17 +13,34 @@ def get_point_cloud_from_bin_file(path_to_bin_file: str) -> o3d.geometry.PointCl
     return point_cloud
 
 
-def write_ones_to_file_by_index_list(
-    point_cloud_size: int, index_list: list, path_to_new_label_file: str
+def generate_labels_and_object_files(
+    point_cloud_size: int,
+    index_list: list,
+    path_to_new_label_file: str,
+    path_to_new_object_file: str,
 ):
     """
     Function changes those lines in the file, the point with the index of which belongs to any plane
     """
-    list_strings = ["0\n"] * point_cloud_size
+    labels_list = ["["] + ["0,"] * point_cloud_size + ["]"]
     for index in index_list:
-        list_strings[index] = "1\n"
-    with open(path_to_new_label_file, "w") as file:
-        file.writelines(list_strings)
+        labels_list[index + 1] = "1,"
```

Comment:
List comprehension?

Reply:
I don't understand what this is about

Reply:
Looks like smth like `labels_list[np.array(index_list) + 1] = "1,"` should work. Also, I don't like the idea to keep labels as strings. Let's separate logic on label values and logic on storage format (that this should be csv string).